### DISCUSSION TITLE
shapleyvalue

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#26] [Announcements]
# shapleyvalue

A quick look at the Shapley value and how I think it's impractical.

https://nonzerosum.games/shapleyvalue.html

**Comment by andrewtaneglen** [comment]
Interesting in terms of how you say these Shapely Value evaluations can be computationally expensive. It is likely due to this that our biases are employed in our behaviour, at the expense of pure reason. Evolutionarily, a less energy-expensive strategy would be favourable in most circumstances, including the energy (and time) expended thinking. So we may have developed a strategy where biases are employed by default under most normal circumstances, to save us having to think things through more thoroughly. There is the argument that people under financial stress suffer a cognitive deficit. There may be an additional evolved quotient to this where people under resource-stress literally dial down their mental capacity to conserve energy. Seems like the opposite argument would more likely apply - that when facing a problem we would dial up our mental capacity to maximise our chances of overcoming whatever we're facing. But under less straining 'normal' situations we might be more likely to fall back on default behaviour biases.

**Reply by jamesstephenbrown** [reply]
Yeah, I agree, you'd think we would evolve to access as much mental capacity as possible in high stress situations - and indeed we can at times "think fast" at times, under particular pressures when we have particular instincts or skills that we can employ relatively automatically. In our evolutionary past, critical thinking in a life and death situation would lead to hesitation, and therefore death.

So, there's that that's probably working against us in times of stress. Regarding cognitive deficit under financial stress -I wonder if testing IQ in the form of an IQ test demonstrates that we are less likely to have the capacity for novel thinking about questions which are essentially irrelevant to us while our brain has certain "worry" cycles on overdrive. And perhaps those "worry" cycles are doing something that might have been useful in our evolutionary past, but aren't quite what we need to solve the complex problems of today.
